packages.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "name": "onnxruntime-nim",
```

Comment:
Names cannot contain dashes, please remove the `-nim`.

Reply:
just removed the dash in name